target_snowflake/__init__.py

```diff
@@ -129,8 +130,28 @@ def persist_lines(config, lines, table_cache=None, file_format_type: FileFormatT
             raise Exception(f"Line is missing required key 'type': {line}")
 
         t = o['type']
-
-        if t == 'RECORD':
+        if t == 'FASTSYNC':
+            
+            stream_row_count = 0 
+            if 'files' in o:
+                stream = o['stream'] 
+
+                for filename in o['files']:
+                    file = os.path.join('fastsync', filename)
+                    size_bytes = os.path.getsize(file)
+                     
+                    with gzip.open(file, 'rb') as f:
+                        for i, l in enumerate(f):
+                            pass
+
+                    count = i + 1
```

Comment:
Is this being used for anything?

Reply:
for logging (i.e Loading X rows into...):

`time=2022-06-07 06:00:16 name=target_snowflake level=INFO message=Loading 250 rows into 'RELEASE_QA."DBO_OAUTHCLIENTPROVIDERS"' `

`time=2022-06-07 06:00:19 name=target_snowflake level=INFO message=Loading 35 rows into 'RELEASE_QA."DBO_OAUTHCLIENTPROVIDERS"'`